### PR TITLE
[AP-801] Add tap-github

### PR DESCRIPTION
**Pull request by koszti**
## Problem

Add tap-github

## Proposed changes

It's using the open source singer tap-github at https://github.com/singer-io/tap-github

## Types of changes

What types of changes does your code introduce to PipelineWise?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Unit tests for changes (not needed for documentation changes)
- [x] CI checks pass with my changes
- [x] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [x] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [x] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: docs/installation_guide/installation.rst [hunk 1]
```diff
@@ -158,7 +162,11 @@ Here’s the list of the singer connectors and if they are installed by default
 +----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
 | tap-zendesk                | ./install --connectors=tap-zendesk      | YES                              |                                       |
 +----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
-| target-postgres            | ./install --connectors=target-postgres  | NO                               |                                       |
+| tap-zuora                  | ./install --connectors=tap-zendesk      | NO                               |                                       |
```
Comment (Samira-El) [inline docs/installation_guide/installation.rst:165]: typo in `--connectors`

File: docs/installation_guide/installation.rst [hunk 2]
```diff
@@ -158,7 +162,11 @@ Here’s the list of the singer connectors and if they are installed by default
...
++----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
+| tap-google-analytics       | ./install --connectors=tap-zendesk      | NO                               |                                       |
```
Comment (Samira-El) [inline docs/installation_guide/installation.rst:167]: typo in `--connectors`

Comment (koszti) [reply docs/installation_guide/installation.rst]: thanks, fixed by https://github.com/transferwise/pipelinewise/pull/473/commits/c960e22a380e4f4bb08997609b292b812f86bb0f

File: docs/installation_guide/installation.rst (hunk 1 above)
Comment (koszti) [reply docs/installation_guide/installation.rst]: thanks, fixed by https://github.com/transferwise/pipelinewise/pull/473/commits/c960e22a380e4f4bb08997609b292b812f86bb0f

**Pull request merged**